### ISSUE TITLE
[Accessibility] trees inside accordion sections should be accessible.

### ISSUE THREAD

**Issue by yzen** [devtools-html/debugger.html#4081]
This is similar and should be consistent with #4067 and #4068. In fact this looks like the same component being reused. If so, it should be fixed in one place so it is accessible (semantically and with the keyboard) everywhere.

### Screenshot

![Screen Shot 2017-09-18 at 8.12.18 PM.png](https://shipusercontent.com/bfbca7328cf6b180bef7f594b4b15d97/Screen%20Shot%202017-09-18%20at%208.12.18%20PM.png)

**Comment by yzen** [comment]
The same solution as #4067 should do the trick. Updating to the latest devtools-core tree component should fix all keyboard and semantics issues, granted https://github.com/devtools-html/devtools-core/issues/695 is fixed first..

**Comment by Garbee** [comment]
This is indeed a duplicate of both the previously mentioned issues. IMO best to close this out to focus on the issues with direction resolution actions instead of using the open space for a filler issue.
